src/wallet/api/walletApi.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { axiosInstance } from './walletServiceAxios';
+import Output from '../../models/output';
+import Input from '../../models/input';
+
+/**
+ * Api calls for wallet
+ *
+ * @namespace ApiWallet
+ */
+
+const walletApi = {
+  getWalletStatus(id: string) {
```

Comment:
Shouldn't we put the type of the expected return? My comment relates to all methods in this file.

Reply:
I've added `Promise<any>` to all of the methods but in the new branch of the signature methods I've created new types for each API return and will add to this file correctly when I continue that branch

Reply:
Ok. Thanks!